modules/dynamic-plugins/rhdh-supported-plugins.template.adoc

```diff
@@ -0,0 +1,59 @@
+[id="rhdh-supported-plugins"]
+= Preinstalled dynamic plugin descriptions and details
+
+// This page is generated! Do not edit the .adoc file, but instead run rhdh-supported-plugins.sh to regen this page from the latest plugin metadata.
+// cd /path/to/rhdh-documentation; ./modules/dynamic-plugins/rhdh-supported-plugins.sh; ./build/scripts/build.sh; google-chrome titles-generated/main/plugin-rhdh/index.html
+
+[IMPORTANT]
+====
+Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend using them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information on Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Scope].
+
+Additional detail on how Red Hat provides support for bundled community dynamic plugins is available on the https://access.redhat.com/policy/developerhub-support-policy[Red Hat Developer Support Policy] page.
+====
+
+There are 61 plugins available in {product}. See the following table for more information:
+
+== Red Hat / Partner provided plugins
```

Comment:
I'm not sure if this is the correct header indicator -- let me know what you want instead by adding commits to this file or making suggestions with the `Add a suggestion` button (CTRL-G)

Reply:
Heading indicator is fine but the header has been updated ^^^